com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -287,20 +289,29 @@ public static void NetworkShow(List<NetworkObject> networkObjects, ulong clientI
                 }
             }
 
-            using (var buffer = PooledNetworkBuffer.Get())
-            using (var writer = PooledNetworkWriter.Get(buffer))
-            {
-                writer.WriteUInt16Packed((ushort)networkObjects.Count);
 
-                for (int i = 0; i < networkObjects.Count; i++)
+            var context = networkManager.MessageQueueContainer.EnterInternalCommandContext(
+                MessageQueueContainer.MessageType.CreateObjects,
+                NetworkChannel.Internal,
+                new[] {clientId},
+                NetworkUpdateLoop.UpdateStage
+            );
+
+            if (context != null)
+            {
+                using (var icontext = (InternalCommandContext) context)
                 {
-                    // Send spawn call
-                    networkObjects[i].Observers.Add(clientId);
+                    icontext.NetworkWriter.WriteUInt16Packed((ushort) networkObjects.Count);
 
-                    networkManager.SpawnManager.WriteSpawnCallForObject(buffer, clientId, networkObjects[i], payload);
-                }
+                    for (int i = 0; i < networkObjects.Count; i++)
+                    {
+                        // Send spawn call
```

Comment:
I'd remove the comment while you're in here.

Reply:
Done